pkgs/applications/misc/bazecor/default.nix

```diff
@@ -39,6 +39,12 @@ appimageTools.wrapAppImage rec {
   extraInstallCommands = ''
     mv $out/bin/bazecor-* $out/bin/bazecor
 
+    install -m 444 -D ${src}/Bazecor.desktop -t $out/share/applications
+    substituteInPlace $out/share/applications/Bazecor.desktop \
+      --replace 'Exec=Bazecor' 'Exec=bazecor'
+
+    install -m 444 -D ${src}/bazecor.png -t $out/share/pixmaps
```

Comment:
Is explicitly stating `${src}` required? It's definitely not _too_ verbose, but I don't think it's needed.

Reply:
It wouldn't be needed if this were a usual `stdenv.mkDerivation`, but this is in a `appimageTools.wrapImage`, and `extraInstallCommands` (via `build-fhsenv-chroot`) is not executed in a context where the cwd contains the `src`, so it is in fact necessary here :+1:

Reply:
Right. I did see that but had a bias since I have packaged using `appimageTools` myself but the only things I used in `extraInstallcommands` were things from the ENV so it was explicit there anyway. This is fine, then :D